mozfun/metadata/normalize/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+description: Accepts a pipeline metadata struct as input and returns a modified struct
```

Comment:
What would you think about a `norm` namespace? So this would become `mozfun.norm.metadata` and then this would be a natural grouping for `norm.os`, `norm.os_version`, etc. as we need for other normalization functions.